backend/internal/storage/postgres/schema/user.go

```diff
@@ -178,14 +206,22 @@ func (r *UserRepository) UpdateUserOnboard(ctx context.Context, email string, en
 	return enthusiasm, nil
 }
 
-func (r *UserRepository) GetUserProfile(ctx context.Context, id uuid.UUID) (*models.Profile, error) {
+func (r *UserRepository) GetUserProfile(ctx context.Context, id uuid.UUID, idType string) (*models.Profile, error) {
```

Comment:
validate `idType` prior to use? or let query result handle invalid?

Reply:
I realized that I should likely just have 2 endpoints -- one for querying profile via ID and another that combines user/display names. It also went over my head that I wanted a list of profiles that are similar or equal to the parameter and not just a single profile. Will commit refactor asap.